x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals/open_close_signals_route.ts

```diff
@@ -57,20 +57,23 @@ export const setSignalsStatusRoute = (router: SecuritySolutionPluginRouter) => {
         };
       }
       try {
-        const result = await clusterClient.callAsCurrentUser('updateByQuery', {
+        const { body } = await esClient.updateByQuery({
           index: siemClient.getSignalsIndex(),
           conflicts: conflicts ?? 'abort',
-          refresh: 'wait_for',
+          // https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-update-by-query.html#_refreshing_shards_2
+          // Note: Before we tried to use "refresh: wait_for" but I do not think that was available and instead it defaulted to "refresh: true"
+          // but the tests do not pass with "refresh: false". If t some point a "refresh: wait_for" is implemented, we should use that instead.
+          refresh: true,
```

Comment:
Thank you for adding context here, appreciate it! 💚  Small typo btw...

```suggestion
          // https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-update-by-query.html#_refreshing_shards_2
          // Note: Before we tried to use "refresh: wait_for" but I do not think that was available and instead it defaulted to "refresh: true"
          // but the tests do not pass with "refresh: false". If at some point a "refresh: wait_for" is implemented, we should use that instead.
          refresh: true,
```